src/popup/utils/constants.js

```diff
@@ -76,7 +76,8 @@ export const defaultNetworks = [
     backendUrl: 'https://test-tipping.aeternity.art',
     tokenRegistry: 'ct_UAzV9RcXEMsFcUCmrPN4iphbZroM7EHk3wvdidDYgZGGBo3hV',
     tokenRegistryLima: 'ct_Dnwribmd21YrxSQnqXCB5vTFPrgYJx2eg2TrbLvbdyEbTMejw',
-    tipContract: 'ct_YpQpntd6fi6r3VXnGW7vJiwPYtiKvutUDY35L4PiqkbKEVRqj',
+    tipContractV1: 'ct_2Cvbf3NYZ5DLoaNYAU71t67DdXLHeSXhodkSNifhgd7Xsw28Xd',
```

Comment:
Why is this different?

Reply:
It is mirroring the superhero ui configuration for https://test-tipping.aeternity.art. Should we change it back?

Reply:
It's the Testnet, so it doesn't matter, but I was just wondering what is the reason.